sql/2020/17_JAMstack/median_lighthouse_score.sql

```diff
@@ -0,0 +1,24 @@
+#standardSQL
+# Lighthouse category scores per SSG
+SELECT
+  _TABLE_SUFFIX AS client,
+  app AS ssg,
+  COUNT(DISTINCT url) AS freq,
+  # See https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087#issuecomment-684983795
+  # APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.performance.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_performance,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.accessibility.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_accessibility,
+  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.pwa.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_pwa
+FROM
+  `httparchive.lighthouse.2020_08_01_*`
+JOIN
+  `httparchive.technologies.2020_08_01_*`
+USING
+  (_TABLE_SUFFIX, url)
+WHERE
+  LOWER(category) = "static site generator" OR
+  app = "Next.js"
+GROUP BY
+  ssg,
+  client
+ORDER BY
+  freq DESC
```

Comment:
We now have LH performance data in the September dataset, so let's update to that.

```suggestion
#standardSQL
# Lighthouse category scores per SSG
SELECT
  _TABLE_SUFFIX AS client,
  app AS ssg,
  COUNT(DISTINCT url) AS freq,
  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.performance.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_performance,
  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.accessibility.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_accessibility,
  APPROX_QUANTILES(CAST(JSON_EXTRACT_SCALAR(report, '$.categories.pwa.score') AS NUMERIC), 1000)[OFFSET(500)] AS median_pwa
FROM
  `httparchive.lighthouse.2020_09_01_*`
JOIN
  `httparchive.technologies.2020_09_01_*`
USING
  (_TABLE_SUFFIX, url)
WHERE
  LOWER(category) = "static site generator" OR
  app = "Next.js"
GROUP BY
  ssg,
  client
ORDER BY
  freq DESC
```

Reply:
Thanks